File: api-gateway/src/main/java/com/research/apigw/ApiGatewayApplication.java
package com.research.apigw;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.data.web.SpringDataWebAutoConfiguration;

//@EnableEurekaClient
@SpringBootApplication(exclude = {org.springframework.boot.autoconfigure.gson.GsonAutoConfiguration.class, SpringDataWebAutoConfiguration.class})
public class ApiGatewayApplication {
    public static void main(String[] args) {
        SpringApplication.run(ApiGatewayApplication.class, args);
    }
}

File: api-gateway/src/main/kotlin/apigw/ApiGatewayApplication.kt
package apigw

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.data.web.SpringDataWebAutoConfiguration
import org.springframework.boot.autoconfigure.gson.GsonAutoConfiguration

@SpringBootApplication(exclude = [GsonAutoConfiguration::class, SpringDataWebAutoConfiguration::class])
open class ApiGatewayApplication

fun main(args: Array<String>) {
    SpringApplication.run(ApiGatewayApplication::class.java, *args)
}